README.md

```diff
@@ -4,26 +4,41 @@ A guide for promoting best practices among RTS code repositories.
 
 ## Introduction
 
-The software industry is constantly evolving. Our own practices should similarly evolve to disseminate knowledge throughout teams, share ownership, make it easier for newcomers to start and for seasoned developers to contribute to any project.
+The software industry is constantly evolving. Our own practices should similarly evolve to
+disseminate knowledge throughout teams, share ownership, make it easier for newcomers to start and
+for seasoned developers to contribute to any project.
 
 ## Guides
 
 The following guides are offered to help you improve your repository:
 
 - [Readme Guide](docs/guides/README_GUIDE.md): Write a great introductory text for your repository.
-- [Contributing Guide](docs/guides/CONTRIBUTING_GUIDE.md): Ensure smooth collaboration and contribution to your project.
-- [Code of Conduct Guide](docs/guides/CODE_OF_CONDUCT_GUIDE.md): Provide guidelines to ensure everyone can safely contribute.
+- [Contributing Guide](docs/guides/CONTRIBUTING_GUIDE.md): Ensure smooth collaboration and
+  contribution to your project.
+- [Code of Conduct Guide](docs/guides/CODE_OF_CONDUCT_GUIDE.md): Provide guidelines to ensure
+  everyone can safely contribute.
 - [License Guide](docs/guides/LICENSE_GUIDE.md): Pick an appropriate license.
 
 > [!TIP]
-> GitHub Community Standards are a great source of information and best practices. You can check that your project follows these standards at any time by checking _Community Standards_ under the _Insights_ section.
+> GitHub Community Standards are a great source of information and best practices. You can check
+> that your project follows these standards at any time by checking _Community Standards_ under the
+_Insights_ section.
 
-More sources are available online for further reading; you can start exploring them at [opensource.guide](https://opensource.guide/).
+More sources are available online for further reading; you can start exploring them
+at [opensource.guide](https://opensource.guide/).
 
 ## Contributing
 
-We welcome contributions from everyone. To get started, please read our [Contribution Guidelines](docs/CONTRIBUTING.md). Feel free to [open issues](https://github.com/SRGSSR/guilde-plateformes-propres/issues/new), [submit pull requests](https://github.com/SRGSSR/guilde-plateformes-propres/compare), or [share your ideas](https://github.com/SRGSSR/guilde-plateformes-propres/discussions).
+We welcome contributions from everyone. To get started, please read
+our [Contribution Guidelines](docs/CONTRIBUTING.md). Feel free
+to [open issues][open-issues], [submit pull requests][submit-pr], or [share your ideas][dicussions].
 
 ## License
 
 See the [LICENSE](LICENSE) file for more information.
+
+[open-issues]: https://github.com/SRGSSR/guilde-plateformes-propres/issues/new
+
+[submit-pr]: https://github.com/SRGSSR/guilde-plateformes-propres/compare
+
+[discussions]: https://github.com/SRGSSR/guilde-plateformes-propres/discussions
```

Comment:
What are these for? Where do they appear?

Reply:
Oh, I see, you extracted the links. Just out of curiosity, what are the benefits of this approach?

Reply:
These are [Reference Style Links][1]. For example this comment is formatted like this:


```markdown
These are [Reference Style Links][1].

[1]: https://www.markdownguide.org/basic-syntax/#reference-style-links
```

You can shorten the link length, it makes it easier to read the raw markdown document.

[1]: https://www.markdownguide.org/basic-syntax/#reference-style-links